docs/appendices/release-notes/unreleased.rst

```diff
@@ -86,6 +86,11 @@ Changes
 
 - Added support of ``GROUP BY`` on ``ARRAY`` typed columns.
 
+- Added support for ``'YES'``, ``'ON'`` and ``'1'`` as alternative way to
```

Comment:
If I'm not mistaken, we add new entries at the beginning of the least, no?

Reply:
🙈  .... moved ✅

Reply:
@matriv We've loosen this rule some time ago as the order does not create much value and the release notes will be grouped and re-arranged before creating a release anyhow.

Reply:
Thx, I didn't know!